cmd/ctr/events.go

```diff
@@ -2,45 +2,75 @@ package main
 
 import (
 	gocontext "context"
+	"encoding/json"
 	"fmt"
 	"os"
 	"text/tabwriter"
 
-	"github.com/containerd/containerd/api/services/execution"
+	eventsapi "github.com/containerd/containerd/api/services/events"
 	"github.com/urfave/cli"
 )
 
 var eventsCommand = cli.Command{
 	Name:  "events",
 	Usage: "display containerd events",
 	Action: func(context *cli.Context) error {
-		tasks, err := getTasksService(context)
+		eventsClient, err := getEventsService(context)
 		if err != nil {
 			return err
 		}
-		events, err := tasks.Events(gocontext.Background(), &execution.EventsRequest{})
+		events, err := eventsClient.EventStream(gocontext.Background(), &eventsapi.EventStreamRequest{})
 		if err != nil {
 			return err
 		}
 		w := tabwriter.NewWriter(os.Stdout, 10, 1, 3, ' ', 0)
-		fmt.Fprintln(w, "TYPE\tID\tPID\tEXIT_STATUS")
 		for {
 			e, err := events.Recv()
 			if err != nil {
 				return err
 			}
+
+			event := ""
+			switch e.Event.TypeUrl {
```

Comment:
The only thing I could find was https://github.com/containerd/containerd/blob/master/services/containers/helpers.go#L30.  But that just specifies the `TypeUrl` as the version which is then the same for everything.  Am I missing something?